### PR TITLE
fix a bug in basicblock

### PR DESCRIPTION
**Pull request by liuzili97**
the stride of conv2 in basic block should be 1

### CODE REVIEW

**Comment by hellock** [comment]
Thanks for the fix! You may also need to remove the padding and dilation of conv2.

File: mmdet/models/backbones/resnet.py [hunk 1]
```diff
@@ -46,9 +46,9 @@ def __init__(self,
             planes,
             planes,
             3,
-            stride=stride,
-            padding=dilation,
-            dilation=dilation,
+            stride=1,
+            padding=1,
```
Comment (hellock) [inline mmdet/models/backbones/resnet.py]: `padding` should be 0 instead. Actually you can just remove padding and dilation to use default values.

Comment (liuzili97) [reply mmdet/models/backbones/resnet.py]: the kernel size of conv2 is 3, so i suppose the padding should be 1.

Comment (hellock) [reply mmdet/models/backbones/resnet.py]: Oh I mistook the kernel size. It should be fine.

> the kernel size of conv2 is 3, so i suppose the padding should be 1.

**Comment by openmmlab-v** [comment]
Hi @liuzili97 ！First of all, we want to express our gratitude for your significant PR in the mmdetection project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Comment by openmmlab-v** [comment]
Hi @liuzili97 ！First of all, we want to express our gratitude for your significant PR in the mmdetection project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**